deploy.sh

```diff
@@ -107,7 +109,7 @@ function main {
         shift
         ;;
       -r|--remote)
-        if [[ -z $2 ]]; then
+        if [[ -z ${2} ]]; then
```

Comment:
I suspect this might fail. The curly braces syntax was meant for use within strings, and this isn't in a string. Variables don't need to be quoted in `[[` conditions because... it magically prevents string splitting somehow I guess.